core/domain/email_manager.py

```diff
@@ -543,3 +545,62 @@ def send_flag_exploration_email(
             recipient_id, feconf.SYSTEM_COMMITTER_ID,
             feconf.EMAIL_INTENT_REPORT_BAD_CONTENT,
             email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_query_completion_email(recipient_id, query_id):
+    email_subject = 'Query %s has successully completed' % query_id
+
+    email_body_template = (
+        'Hi %s,<br>'
+        'Your query with %s id has succesfully completed its '
+        'execution. Visit the result page '
+        '<a href="https://www.oppia.org/emaildashboardresult/%s">here</a> '
+        'to see result of your query.<br><br>'
+        'Thanks, and happy collaborating!<br>'
```

Comment:
I don't think you need the "and happy collaborating" part, since the user isn't actually collaborating much in this scenario :)